Objects/typeobject.c

```diff
@@ -2965,27 +2967,20 @@ _PyType_Lookup(PyTypeObject *type, PyObject *name)
 
     if (mro == NULL) {
         if ((type->tp_flags & Py_TPFLAGS_READYING) == 0 &&
-            PyType_Ready(type) < 0) {
-            /* It's not ideal to clear the error condition,
-               but this function is documented as not setting
-               an exception, and I don't want to change that.
-               When PyType_Ready() can't proceed, it won't
-               set the "ready" flag, so future attempts to ready
-               the same type will call it again -- hopefully
-               in a context that propagates the exception out.
-            */
-            PyErr_Clear();
+                PyType_Ready(type) < 0) {
```

Comment:
Either move `PyType_Ready()` to the previous line (if the resulting line will be not too long), or indent it as in the current code and move `{` to a new line for readability (new PEP 7 rule).

Reply:
I cleaned it up a little.